Formula/gnu-getopt.rb

```diff
@@ -1,31 +1,34 @@
+# This formula was previously based on the project hosted by the original
+# developer, which has since been merged into util-linux. The util-linux formula
+# could not be repurposed to provide getopt, though, since it only includes
+# tools that aren't already shipped with macOS. Thus, we now have two formulae
+# that offer independent subsets of util-linux
 class GnuGetopt < Formula
-  desc "Command-line option parsing library"
-  homepage "http://frodo.looijaard.name/project/getopt"
-  url "http://frodo.looijaard.name/system/files/software/getopt/getopt-1.1.6.tar.gz"
-  mirror "https://distfiles.macports.org/getopt/getopt-1.1.6.tar.gz"
-  mirror "https://fossies.org/linux/misc/getopt-1.1.6.tar.gz"
-  sha256 "d0bf1dc642a993e7388a1cddfb9409bed375c21d5278056ccca3a0acd09dc5fe"
+  desc "Command-line option parsing utility"
+  homepage "https://github.com/karelzak/util-linux"
+  url "https://www.kernel.org/pub/linux/utils/util-linux/v2.32/util-linux-2.32.1.tar.xz"
+  sha256 "86e6707a379c7ff5489c218cfaf1e3464b0b95acf7817db0bc5f179e356a67b2"
 
   bottle do
+    cellar :any
     sha256 "e905a353b1ef1688e569ee28f5caa35bbfa6a4b99f044e255087d0a8adbe092a" => :mojave
     sha256 "5dc8b07eb3425e5b57d7deb4dea187fc992ef358c9c053d3a2dc230d748b4252" => :high_sierra
     sha256 "05391b0dd0876ead74b18a4c5bb9c7db996586bf6918bd014db534027fd9ae2a" => :sierra
     sha256 "5e9e87fe18c5681e80f1cf940fed275ed895831304326bc5e7be6fb6e53e8594" => :el_capitan
-    sha256 "f8dbbec03aaaeb1bc774d9bf606701901cc9a8ad15cecc5473567e51845057e6" => :yosemite
-    sha256 "27938c615808c8e4ff2eacac0a4059c76dee5518a5c8bbfb304b24b70736b429" => :mavericks
```

Comment:
Please don't edit the `bottle do` block. This gets updated manually during the CI build process. If it needs adjustment after that it is the responsibility of maintainers to take care of that.